### PR TITLE
MOM 6 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/dusk_legion_duelist.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Dusk Legion Duelist
+ManaCost:1 W
+Types:Creature Vampire Soldier
+PT:2/2
+K:Vigilance
+T:Mode$ CounterAddedOnce | ValidCard$ Card.Self | ActivationLimit$ 1 | TriggerZones$ Battlefield | CounterType$ P1P1 | Execute$ TrigDraw | TriggerDescription$ Whenever one or more +1/+1 counters are put on CARDNAME, draw a card. This ability triggers only once each turn.
+
+
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/dusk_legion_duelist.txt:8]: delete empty lines?

File: forge-gui/res/cardsfolder/upcoming/bloodfeather_phoenix.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Bloodfeather Phoenix
+ManaCost:1 R
+Types:Creature Phoenix
+PT:2/2
+K:Flying
+K:CARDNAME can't block.
+T:Mode$ DamageDone | ValidSource$ Instant.YouCtrl,Sorcery.YouCtrl | CombatDamage$ False | ValidTarget$ Opponent,Battle | TriggerZones$ Graveyard | Execute$ TrigReturn | TriggerDescription$ Whenever an instant or sorcery spell you control deals damage to an opponent or battle, you may pay {R}. If you do, return CARDNAME from your graveyard to the battlefield. It gains haste until end of turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bloodfeather_phoenix.txt:7]: Remove CombatDamage$

File: forge-gui/res/cardsfolder/upcoming/axgard_artisan.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Axgard Artisan
+ManaCost:1 R
+Types:Creature Dwarf Artificer
+PT:2/1
+T:Mode$ CounterAddedOnce | ValidCard$ Card.Self | ActivationLimit$ 1 | TriggerZones$ Battlefield | CounterType$ P1P1 | Execute$ TrigToken | TriggerDescription$ Whenever one or more +1/+1 counters are put on CARDNAME for the first time each turn, create a Treasure token. (it's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")
```
[the remark this answers is not in the document]
Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/axgard_artisan.txt:5]: might be related to this: #2784

[the remark this answers is not in the document]
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/axgard_artisan.txt:5]: Thanks seems to work in testing

[the remark this answers is not in the document]
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/axgard_artisan.txt]: No, you need to ask yourself why it only works with LE1
Then you could test something like _Dragonscale Boon_ and see that it probably won't work
In that case it means the idea made a wrong assumption and we might need to pass the info as AbilityKey instead

[the remark this answers is not in the document]
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/axgard_artisan.txt]: Yeah You're right, doesn't seem to work with Dragonscale Boon. I'll remove the card from this PR

File: forge-gui/res/cardsfolder/upcoming/deeproot_wayfinder.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Name:Deeproot Wayfinder
+ManaCost:1 G
+Types:Creature Merfolk Scout
+PT:2/3
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player,Battle | CombatDamage$ True | Execute$ TrigSurveil | TriggerDescription$ Whenever CARDNAME deals combat damage to a player or battle, surveil 1. (To surveil 1, look at the top card of your library. You may put that card into your graveyard.)
+SVar:TrigSurveil:DB$ Surveil | Amount$ 1 | SubAbility$ DBChangeZone | SpellDescription$ Then you may return a land card from your graveyard to the battlefield tapped.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/deeproot_wayfinder.txt:6]: TriggerDescription done wrong / incomplete

**Review by tool4ever — APPROVED** [review]

**Pull request merged**